### PR TITLE
add endpoint GetMakeOfferByCardID that returns make offer data

### PR DESCRIPTION
**Pull request by APrisiazhniuk**

### CODE REVIEW

File: marketplace/marketplace.go [hunk 1]
```diff
@@ -104,8 +105,9 @@ type Config struct {
 	MarketplaceNFTContract struct {
 		Address string `json:"address"`
 	} `json:"marketplaceNFTContract"`
-	RPCNodeAddress string `json:"rpcNodeAddress"`
-	ContractHash   string `json:"contractHash"`
+	RPCNodeAddress      string                `json:"rpcNodeAddress"`
+	ContractHash        string                `json:"contractHash"`
```
Comment (Dmytro-yakymuk) [inline marketplace/marketplace.go]: nft or market contract?

File: marketplace/marketplace.go [hunk 2]
```diff
@@ -104,8 +105,9 @@ type Config struct {
...
+	CasperTokenContract evmsignature.Contract `json:"casperTokenContract"`
```
Comment (Dmytro-yakymuk) [inline marketplace/marketplace.go]: casper contract has type from evmsignature?

File: console/consoleserver/server.go [hunk 3]
```diff
@@ -191,6 +191,7 @@ func NewServer(config Config, log logger.Logger, listener net.Listener, cards *c
 
 	bidsRouter := apiRouter.PathPrefix("/bids").Subrouter()
 	bidsRouter.Use(server.withAuth)
+	bidsRouter.HandleFunc("/make-offer/{card_id}", bidsController.GetMakeOfferData).Methods(http.MethodGet)
```
Comment (Dmytro-yakymuk) [inline console/consoleserver/server.go]: mb /make-offer/{card_id} -> /offer/{card_id}
it's just get method

File: marketplace/marketplace.go (hunk 1 above)
Comment (APrisiazhniuk) [reply marketplace/marketplace.go]: it is old nft contract - we need it for claiming NFT

File: marketplace/marketplace.go (hunk 2 above)
Comment (APrisiazhniuk) [reply marketplace/marketplace.go]: redone to string

File: console/consoleserver/server.go (hunk 3 above)
Comment (APrisiazhniuk) [reply console/consoleserver/server.go]: +

**Pull request merged**